packaging_automation/tests/test_docker_statistics_collector.py

```diff
@@ -0,0 +1,34 @@
+from ..docker_statistics_collector import fetch_and_store_docker_statistics, DockerStats
+
+from datetime import datetime, timedelta
+from ..dbconfig import Base, db_connection_string
+from sqlalchemy import create_engine, text, engine
+from sqlalchemy.orm import sessionmaker
+
+
+def test_docker_statistics_collector():
+    test_day_shift_index = 2
+    test_pull_count_shift = 200
+    db = create_engine(db_connection_string(is_test=True))
+    sql = text(f'DROP TABLE {DockerStats.__tablename__}')
+    db.execute(sql)
+    fetch_and_store_docker_statistics("citus", is_test=True)
+    first_day = datetime.today()
```

Comment:
Can you move this next to second and third day

Reply:
Fixed